developers/webnative/platform.md

```diff
@@ -44,9 +44,27 @@ const newApp = await sdk.apps.create()
 // 'your-fission-deployment.fission.app'
 ```
 
-**apps.deleteByURL**
+**apps.publish**
 
-Destroy app by any associated URL
+Publishes a new app version by IPFS CID.
+
+Params:
+
+* domain: `string` **required**
+* cid: `string` **required**
+
+Example:
+
+```typescript
+await sdk.apps.publish('your-fission-deployment.fission.app', 'QmRVvvMeMEPi1zerpXYH9df3ATdzuB63R1wf3Mz5NS5HQN')
+```
+
+Getting a CID can be tricky. Here's a way to turn a WNFS public subdirectory into a CID:
+https://github.com/fission-suite/dashboard/blob/8ef6e7031b8f6f20f2927e7e6c16349d9ba24ec9/src/Javascript/index.js#L334-L339
```

Comment:
Wondering if we may want to inline this example code into the docs? It's not really what this section is about, but we can give it more context by putting it here. Like, what is `appPath`? Is the app a public subdirectory?

Reply:
Yup sounds like a good idea!